hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/HFileBlock.java

```diff
@@ -1729,7 +1729,31 @@ protected HFileBlock readBlockDataInternal(FSDataInputStream is, long offset,
         }
         onDiskSizeWithHeader = getOnDiskSizeWithHeader(headerBuf, checksumSupport);
       }
-      int preReadHeaderSize = headerBuf == null ? 0 : hdrSize;
+      int preReadHeaderSize = 0;
+      if (
+        headerBuf == null
+          || headerBuf.getInt(Header.ON_DISK_SIZE_WITHOUT_HEADER_INDEX) != onDiskSizeWithHeader
+      ) {
```

Comment:
the braces's style is not right?

Reply:
The style is modified by spotless automatically after I ran spotless:apply